examples/dpgen2/build.envd

```diff
@@ -0,0 +1,40 @@
+def install_kubectl_kind():
```

Comment:
Should we keep it in envd examples or envdlib?

Reply:
It's not general enough now. `sudo docker network connect kind dpgen2` means only project named as dpgen2 will work

Reply:
Could we make like this:

```
def install_kubectl_kind(environment_name):
```

Reply:
Changed to `def install_kubectl_kind(env_name)`. But I still think `kind` is not a general requirement. Thus I don't think we don't need to put it into envdlib